lib/ansible/plugins/netconf/junos.py

```diff
@@ -152,3 +152,39 @@ def reboot(self):
     def halt(self):
         """reboot the device"""
         return self.m.halt().data_xml
+
+    @ensure_connected
+    def get(self, *args, **kwargs):
+        try:
+            return self.m.get(*args, **kwargs).data_xml
+        except RPCError as exc:
+            raise Exception(to_xml(exc.xml))
+
+    @ensure_connected
+    def get_config(self, *args, **kwargs):
```

Comment:
Why is this changed required for junos plugin?

Reply:
abstracted out base class methods to handle platform specific behaviour.